### PR TITLE
Added test files for Review and ReviewManager

### PR DESCRIPTION
**Pull request by guybeckenstein**
#50 
## Summary
- Fixed **str()** function in _models.py_ (_review.models.Review_). Also changed the documentation in this file
- Modified migration files _0001_initial.py_ and _0002_test_data.py_, mainly added test data
- Created test files _test_review.py_ and _test_review_manager.py_.
  - 3 tests for **models.py** class: _Review_
  - 4 tests for **models.py** class: _ReviewManager_

### CODE REVIEW

**Review by idosinger98 — APPROVED** [review]

**Review by patrisiakaplun — APPROVED** [review]
LGTM

File: review/tests/test_review.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import pytest
+from datetime import timedelta
+from django.utils import timezone
+from review.models import Review
+
+
+@pytest.fixture
+def review():
```
Comment (Yarboa) [inline review/tests/test_review.py:8]: The fixture, should create review or reviews objects
Why there is logic here? 

If there is a logic, you should test it, is'nt it?

File: review/tests/test_review.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    # Checks if there are at least one object of the same Review and then deletes it
+    if len(review) > 0:
+        review.delete()
+    # Creates and returns a new review object
+    now = timezone.now()
+    review = Review.objects.create(rating='4', description='Creating a test review...',
+                                   date_posted=now - timedelta(days=50),
+                                   client_id='client_id', professional_id='professional_id')
+    yield review        # Returns review object
+    review.delete()     # TearDown
```
Comment (Yarboa) [inline review/tests/test_review.py:20]: This is volatile data, you can avoid the delete

**Review by idosinger98 — COMMENTED** [review]
Why do you need two test files?

File: review/tests/test_review.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+@pytest.mark.django_db
+def test_method_filter_by_professional_id(review):
+    filtered_reviews = Review.filter_by_professional_id(professional_id="professional_id")
+    filtered_reviews_lst = list(filtered_reviews)
+
+    try:
+        from django.template.backends import django
```
Comment (Yarboa) [inline review/tests/test_review.py:29]: move to head of the file

File: review/tests/test_review.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        if type(review) is Review:
+            assert filtered_reviews_lst == [review]
+        elif type(review) is django.db.models.query.QuerySet:
+            assert filtered_reviews_lst == [*review]
+        else:
+            raise TypeError
```
Comment (Yarboa) [inline review/tests/test_review.py:35]: you can assert error assertion 
 with pytest.raises(Exception) as TypeError

File: review/tests/test_review_manager.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
+import pytest
+from datetime import timedelta
+from django.utils import timezone
+from review.models import Review
+
+
+@pytest.fixture
+def reviews():
+    now = timezone.now()
```
Comment (Yarboa) [inline review/tests/test_review_manager.py:9]: once migration data is changes that test will fail
Note, did you think of it?

File: review/tests/test_review_manager.py [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+                                 client_id='user4', professional_id='pro1')
+    review2 = get_single_reviews(review2, rating='4', description='Good', date_posted=now - timedelta(days=1),
+                                 client_id='user5', professional_id='pro2')
+    review3 = get_single_reviews(review3, rating='3', description='Average', date_posted=now - timedelta(days=2),
+                                 client_id='user6', professional_id='pro3')
+    yield
+    tear_down(review1, review2, review3)
+
+
+def get_single_reviews(review, rating, description, date_posted, client_id, professional_id):
```
Comment (Yarboa) [inline review/tests/test_review_manager.py:25]: Is there a better way to test get single review?

File: review/tests/test_review.py (hunk 1 above)
Comment (guybeckenstein) [reply review/tests/test_review.py:9]: Fixed

File: review/tests/test_review.py (hunk 2 above)
Comment (guybeckenstein) [reply review/tests/test_review.py]: > This is volatile data, you can avoid the delete

Fixed

File: review/tests/test_review.py (hunk 3 above)
Comment (guybeckenstein) [reply review/tests/test_review.py]: > move to head of the file

Fixed

File: review/tests/test_review.py (hunk 4 above)
Comment (guybeckenstein) [reply review/tests/test_review.py]: > you can assert error assertion

Added

File: review/tests/test_review_manager.py (hunk 5 above)
Comment (guybeckenstein) [reply review/tests/test_review_manager.py:9]: > once migration data is changes that test will fail Note, did you think of it?

Should I remove migration data?

File: review/tests/test_review_manager.py (hunk 6 above)
Comment (guybeckenstein) [reply review/tests/test_review_manager.py]: > Is there a better way to test get single review?

Fixed

File: review/tests/test_review_manager.py (hunk 5 above)
Comment (Yarboa) [reply review/tests/test_review_manager.py:9]: It depends,
Anyway be aware it could cause failures in your tests if migration dat is changed

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**